app/assets/javascripts/admin/spree/taxons/taxonomy.js.coffee

```diff
@@ -40,8 +40,8 @@ handle_rename = (e, data) ->
   node = data.rslt.obj
   name = data.rslt.new_name
 
-  url = Spree.url(base_url).clone()
-  url.pathname = url.pathname + '/' + node.attr("id")
+  url =  $.extend(true, {}, base_url)
+  url = url['href'] + '/' + node.attr("id")
```

Comment:
Why did you change that? 
Before we only update the `pathname` attribute of that URL object.
Here we entirely reassign a new object to a String (and therefore the copy the line below is not needed).
I think the previous code is functional.

Reply:
Thank you for the suggestion! I will be sure to make that change to the pathname attribute, instead. I've just committed your request changes for the first suggestions. My team was unable to determine how to rename the first taxon in the list as the input form takes precedent. Handle_delete and handle_move will have similar solutions. I am currently still at university; thus, I have very limited resources. In a few weeks, I should have more availability to fix these issues as well, if they are still open 👍

Reply:
Ok, I've opened an issue then: https://github.com/openfoodfoundation/openfoodnetwork/issues/9113